src/data_structures/findTranslocationsOnTheFly.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+   Francesco Vezzi
+   Jesper Eisfeldt
+ */
+
```

Comment:
Suggestion: as right now you are implementing a new way to spot translocations I would call this file `findTranslocations` and inside it put several possible implementations of similar functions. This will give you the possibility to implement one function without using an hold but working one.

Other suggestion: would be nice to call this file with the same name you employ to call the module in the options. In this way you might be able to call the right module during execution time.